examples/ConstraintIB/flow_past_cylinder_HF/HydroForceEval.cpp

```diff
@@ -132,12 +130,10 @@ HydroForceEval::HydroForceEval(const std::string& object_name,
         d_force_stream.resize(d_num_structs);
         for (int struct_no = 0; struct_no < d_num_structs; ++struct_no)
         {
-            std::ostringstream force_stream;
-            force_stream << d_dir_name + '/' + d_struct_names[struct_no] + "_HF.txt";
             if (from_restart)
-                d_force_stream[struct_no] = new std::ofstream(force_stream.str().c_str(), std::fstream::app);
+                d_force_stream[struct_no] = new std::ofstream((d_dir_name + '/' + d_struct_names[struct_no] + "_HF.txt").c_str(), std::fstream::app);
             else
-                d_force_stream[struct_no] = new std::ofstream(force_stream.str().c_str(), std::fstream::out);
+                d_force_stream[struct_no] = new std::ofstream((d_dir_name + '/' + d_struct_names[struct_no] + "_HF.txt").c_str(), std::fstream::out);
```

Comment:
While we are here: the calls to `c_str()` are no longer necessary.

Reply:
Ok, got it.